File: PauloVitor/GerenciadorVenda.java
public class GerenciadorVenda{

    public Venda novaVenda(float valor, String pagamento, Cliente cliente){
        v.valor = valor;
        v.pagamento = pagamento;
        v.cliente = cliente
        return v;
    }

    public void consultarVenda(Venda v){
        System.out.println(v.valor);
        System.out.println(v.pagamento);
        System.out.println("Cliente associado: ");
        c.consultar(v.cliente);
    }

}

File: verenilson/GerenciadorVenda.cs
import package::GerenciadorVenda;
class GerenciadorVenda{
	String nomeDaVenda;
	float Valor;
	GerenciadorVenda venda = new GerenciadorVenda();
	
	private void chamarMetodo(){
		if(venda.inserirCliente()){
			print("Cliente foi inserido!");
		}
	}
}